src/sync/PostSyncProcessor.js

```diff
@@ -113,7 +108,7 @@ export class PostSyncProcessor {
    * @param  {object} record     The record changed
    * @return {array}  An array of functions to be called for the given record
    */
-  postProcessRequisition(record) {
+  generatePostProcessesForRequisition(record) {
```

Comment:
One more change now I see it... `generateProcessorsForRequisition` 
- It generates functions that have an effect on something, so they're 'processors' more than they are 'processes'
- We know it's post sync, that's the name of the class, so we can probably drop the 'post' bit.)